### PR TITLE
Demonic coup

### PR DESCRIPTION
**Pull request by acourrau**
Added Demonic Coup card and placeholder art for The Cull character.

### CODE REVIEW

File: src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json [hunk 1]
```diff
@@ -168,6 +168,10 @@
     "DESCRIPTION": "stsjorbsmod:Entombed. stsjorbsmod:Ephemeral. NL stsjorbsmod:Exhume whenever an enemy dies. NL Deal !D! damage. NL Remember stsjorbsmod:Wrath.",
     "UPGRADE_DESCRIPTION": "stsjorbsmod:Entombed. stsjorbsmod:Ephemeral. NL stsjorbsmod:Exhume whenever an enemy dies. NL Deal !D! damage. NL Gain stsjorbsmod:Clarity of stsjorbsmod:Wrath."
   },
+  "stsjorbsmod:DemonicCoup": {
+    "NAME": "Demonic Coup",
+    "DESCRIPTION": "Gain !M! Ritual. NL stsjorbsmod:Exert."
```
Comment (dbjorge) [inline src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json:173]: I think this is missing a `stsjorbsmod:Ephemeral. NL ` at the start (the ephemeral effect is implemented correctly, just missing on the text)

Comment (acourrau) [reply src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json:173]: Yup, you're right, I'll add.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 * New art for Thirsting campfire option (thanks ChDaVinci)
 * New art for Withering (thanks ChDaVinci for art and Neablis for helping merge)
 * New art for Decoy (thanks EscMargot for art and Neablis for helping merge)
+* Added Demonic Coup card and placeholder art
```
Comment (dbjorge) [inline CHANGELOG.md:16]: There's a "cards" section a few lines down this should probably go in. Also, would be good if you could credit yourself here :)

Comment (acourrau) [reply CHANGELOG.md:16]: It felt weird saying "Good job me!" I figured someone else could. Will add. I wasn't sure what all needed to be updated in the changelog. Is it everything before the last release tag?

**Pull request merged**